python/ray/util/actor_pool.py

```diff
@@ -10,7 +10,9 @@ class ActorPool:
     Examples:
         >>> a1, a2 = Actor.remote(), Actor.remote()
         >>> pool = ActorPool([a1, a2])
-        >>> print(pool.map(lambda a, v: a.double.remote(v), [1, 2, 3, 4]))
+        >>> results = pool.map(lambda a, v: a.double.remote(v), [1, 2, 3, 4])
+        >>> for r in results:
+        ...     print(r)
```

Comment:
It shouldn't be a big deal, but the output written underneath will be 

```
2
4
6
8
```

instead of [2,4,6,8]. Assuming the pool.map will return iterable, isn't it possible to just run

```python3
print(list(pool,map(lambda a, v: a.double.remote(v), [1, 2, 3, 4])))
```
?

Reply:
Thanks @rkooo567 ! Great suggestions; I updated the patch.